src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -103,7 +122,6 @@ public void storeLocation(Location location) {
     PreparedQuery results = datastore.prepare(query);
 
     for (Entity entity : results.asIterable()) {
-
```

Comment:
This line, for example, should be re-added. You're not modifying any nearby line at all.